### ISSUE TITLE
Problem with ENUM.values()

### ISSUE THREAD

**Issue by tgart** [jtwig/jtwig#267]
Hi, i have a problem with ENUM, when i want add ENUM.values() to template VIEW doesn't display anything (as like this variable is empty). I think is a problem with Array. In Twig 2.x working fine.

Sample CODE:

```
public enum SampleEnum {
    ONE, TWO;
}
```

```
JtwigModelMap jtwigModelMap = new JtwigModelMap();
jtwigModelMap.add("sampleEnums", SampleEnum.values());
```

```
<select>
{% for sampleEnumItem in sampleEnums %}
<option>{{ sampleEnumItem }}</option>
{% endfor %}
</select>
```

and render:
```
<select>
</select>
```

should render:
```
<select>
<option>ONE</option>
<option>TWO</option>
</select>
```

Thanks for your help.

**Comment by thomas-p-wilson** [comment]
Sorry for the wait @tgart. I will look into this matter shortly.

**Comment by thomas-p-wilson** [comment]
Quite right that this is an array issue. I suspect that arrays of any type would have resulted in the same behaviour.

**Comment by thomas-p-wilson** [comment]
Fixed in PR #270 for 4.0.0 release
